packages/office-ui-fabric-react/src/components/ExtendedPicker/BaseExtendedPicker.tsx

```diff
@@ -279,6 +283,19 @@ export class BaseExtendedPicker<T, P extends IBaseExtendedPickerProps<T>> extend
     }
   };
 
+  /**
+   * The floating picker is the source of truth for if the menu has been opened or not.
+   *
+   * Because this isn't tracked inside the state of this component, we need to
+   * force an update here to keep the rendered output that depends on the picker being open
+   * in sync with the state
+   *
+   * Called when the suggestions is shown or closed
+   */
+  private _onSuggestionsShownOrHidden = () => {
```

Comment:
Hi @Adjective-Object , the build's tslint check is failing on this new private method coming before the protected one's. Please place this private method at the bottom of the class or rather just below the last `protected` method, thanks!